src/components/dashboard/DashboardNav.js

```diff
@@ -11,15 +12,41 @@ export default class DashboardNav extends Component {
   constructor(props) {
     super(props);
 
-    this.handleSubmit = this.handleSubmit.bind(this);
+    this.state = {
+      showNewTrip: false
+    };
+
+    this.handleLogout = this.handleLogout.bind(this);
+    this.startCreateEvent = this.startCreateEvent.bind(this);
+    this.quitCreateEvent = this.quitCreateEvent.bind(this);
   }
 
   /***************************** Core functions *****************************/
-  handleSubmit(e) {
+  /**
+   * Logs user out
+   * @param e click event
+   */
+  handleLogout(e) {
     e.preventDefault();
     this.props.onLogout();
   }
 
+  /**
+   * This closes the create trip modal.
+   */
+  quitCreateEvent() {
+    this.setState({ showNewTrip: false });
+  }
+
+  /**
+   * This opens the create trip modal.
+   * @param e click event
+   */
+  startCreateEvent(e) {
+    e.preventDefault();
```

Comment:
Out of curiosity, is there a reason we use preventDefault in some places but don't in others?

Reply:
Tbh I might over use it, but I tend to call it whenever I'm creating a click action. It just keeps the browser from doing anything you haven't directly specified is my understanding